bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZH_ZToNuNu_1j_SMEFTsim_topU3l/ZH_ZToNuNu_1j_SMEFTsim_topU3l_reweight_card.dat

```diff
@@ -0,0 +1,2759 @@
+set cHj1 0.000000
+set cHj3 0.000000
+set cHu 0.000000
+set cHd 0.000000
+set cHudRe 0.000000
+set cHudIm 0.000000
+set cHW 0.000000
+set cHWtil 0.000000
+set cHB 0.000000
+set cHBtil 0.000000
+set cHWB 0.000000
+set cHWBtil 0.000000
+set cHbox 1.000000
+set cHDD 0.000000
+set cbHRe 0.000000
+set cbHIm 1.000000
+
+launch --rwgt_name=cHj1_0p_cHj3_0p_cHu_0p_cHd_0p_cHudRe_0p_cHudIm_0p_cHW_0p_cHWtil_0p_cHB_0p_cHBtil_0p_cHWB_0p_cHWBtil_0p_cHbox_0p_cHDD_2p_cbHRe_0p_cbHIm_0p
+set cHj1 0.000000
+set cHj3 0.000000
+set cHu 0.000000
+set cHd 0.000000
+set cHudRe 0.000000
+set cHudIm 0.000000
+set cHW 0.000000
+set cHWtil 0.000000
+set cHB 0.000000
+set cHBtil 0.000000
+set cHWB 0.000000
+set cHWBtil 0.000000
+set cHbox 0.000000
+set cHDD 2.000000
+set cbHRe 0.000000
+set cbHIm 0.000000
+
+launch --rwgt_name=cHj1_0p_cHj3_0p_cHu_0p_cHd_0p_cHudRe_0p_cHudIm_0p_cHW_0p_cHWtil_0p_cHB_0p_cHBtil_0p_cHWB_0p_cHWBtil_0p_cHbox_0p_cHDD_1p_cbHRe_1p_cbHIm_0p
+set cHj1 0.000000
+set cHj3 0.000000
+set cHu 0.000000
+set cHd 0.000000
+set cHudRe 0.000000
+set cHudIm 0.000000
+set cHW 0.000000
+set cHWtil 0.000000
+set cHB 0.000000
+set cHBtil 0.000000
+set cHWB 0.000000
+set cHWBtil 0.000000
+set cHbox 0.000000
+set cHDD 1.000000
+set cbHRe 1.000000
+set cbHIm 0.000000
+
+launch --rwgt_name=cHj1_0p_cHj3_0p_cHu_0p_cHd_0p_cHudRe_0p_cHudIm_0p_cHW_0p_cHWtil_0p_cHB_0p_cHBtil_0p_cHWB_0p_cHWBtil_0p_cHbox_0p_cHDD_1p_cbHRe_0p_cbHIm_1p
+set cHj1 0.000000
+set cHj3 0.000000
+set cHu 0.000000
+set cHd 0.000000
+set cHudRe 0.000000
+set cHudIm 0.000000
+set cHW 0.000000
+set cHWtil 0.000000
+set cHB 0.000000
+set cHBtil 0.000000
+set cHWB 0.000000
+set cHWBtil 0.000000
+set cHbox 0.000000
+set cHDD 1.000000
+set cbHRe 0.000000
+set cbHIm 1.000000
+
+launch --rwgt_name=cHj1_0p_cHj3_0p_cHu_0p_cHd_0p_cHudRe_0p_cHudIm_0p_cHW_0p_cHWtil_0p_cHB_0p_cHBtil_0p_cHWB_0p_cHWBtil_0p_cHbox_0p_cHDD_0p_cbHRe_2p_cbHIm_0p
+set cHj1 0.000000
+set cHj3 0.000000
+set cHu 0.000000
+set cHd 0.000000
+set cHudRe 0.000000
+set cHudIm 0.000000
+set cHW 0.000000
+set cHWtil 0.000000
+set cHB 0.000000
+set cHBtil 0.000000
+set cHWB 0.000000
+set cHWBtil 0.000000
+set cHbox 0.000000
+set cHDD 0.000000
+set cbHRe 2.000000
+set cbHIm 0.000000
+
+launch --rwgt_name=cHj1_0p_cHj3_0p_cHu_0p_cHd_0p_cHudRe_0p_cHudIm_0p_cHW_0p_cHWtil_0p_cHB_0p_cHBtil_0p_cHWB_0p_cHWBtil_0p_cHbox_0p_cHDD_0p_cbHRe_1p_cbHIm_1p
+set cHj1 0.000000
+set cHj3 0.000000
+set cHu 0.000000
+set cHd 0.000000
+set cHudRe 0.000000
+set cHudIm 0.000000
+set cHW 0.000000
+set cHWtil 0.000000
+set cHB 0.000000
+set cHBtil 0.000000
+set cHWB 0.000000
+set cHWBtil 0.000000
+set cHbox 0.000000
+set cHDD 0.000000
+set cbHRe 1.000000
+set cbHIm 1.000000
+
+launch --rwgt_name=cHj1_0p_cHj3_0p_cHu_0p_cHd_0p_cHudRe_0p_cHudIm_0p_cHW_0p_cHWtil_0p_cHB_0p_cHBtil_0p_cHWB_0p_cHWBtil_0p_cHbox_0p_cHDD_0p_cbHRe_0p_cbHIm_2p
```

Comment:
Actually, it is more explicit right now. There are also combinations where two parameters are turned on. 
In the end, it is just a convention. Since we have the same convention in other existing cards, which were used to produce central samples, we would like to keep it the same. 

[1]
https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WH_WToLNu_1j_SMEFTsim_topU3l/WH_WToLNu_1j_SMEFTsim_topU3l_reweight_card.dat
https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZH_ZToLL_1j_SMEFTsim_topU3l/ZH_ZToLL_1j_SMEFTsim_topU3l_reweight_card.dat